### PR TITLE
Added a new feature

### PR DESCRIPTION
**Pull request by rmabow03**
this feature is for adding and autocomplete into the search box.

### CODE REVIEW

File: lab-exercise-1/RadchaneekornFeature.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+public class RadchaneekornFeature {
+      public void feature() {
+          // this is a new feature
+      }
```
Comment (6588041chalisakengkaewpennapa) [inline lab-exercise-1/RadchaneekornFeature.java:4]: code is empty you should fix

Comment (jaxmmmm) [inline lab-exercise-1/RadchaneekornFeature.java:5]: It looks not complete.

Comment (Napassawxn) [reply lab-exercise-1/RadchaneekornFeature.java:5]: I think you should fix it

Comment (jaxmmmm) [inline lab-exercise-1/RadchaneekornFeature.java:5]: Fix pls

Comment (jaxmmmm) [reply lab-exercise-1/RadchaneekornFeature.java:5]: It empty

Comment (Napassawxn) [reply lab-exercise-1/RadchaneekornFeature.java:5]: fix now

File: lab-exercise-1/RadchaneekornFeature.java [hunk 2]
```diff
@@ -1,6 +1,7 @@
 public class RadchaneekornFeature {
       public void feature() {
-          // this is a new feature
```
Comment (Napassawxn) [inline lab-exercise-1/RadchaneekornFeature.java]: Delete it

File: lab-exercise-1/RadchaneekornFeature.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
+public class RadchaneekornFeature {
+      public void feature() {
+          // this is a new feature
```
Comment (Napassawxn) [inline lab-exercise-1/RadchaneekornFeature.java]: Delete it

File: lab-exercise-1/RadchaneekornFeature.java (hunk 1 above)
Comment (JNapat) [reply lab-exercise-1/RadchaneekornFeature.java:5]: empty

File: lab-exercise-1/RadchaneekornFeature.java (hunk 2 above)
Comment (jaxmmmm) [reply lab-exercise-1/RadchaneekornFeature.java]: improve pls

**Pull request merged**